<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CremittanceSetupDlg</class>
 <widget class="QDialog" name="CremittanceSetupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="m_accountTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>781</width>
     <height>251</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <column>
    <property name="text">
     <string>Account Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Local Share</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hq Share</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Account Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Local Share</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hq Share</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="m_selectAccountLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Account For Femittance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_okBtn">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>380</y>
     <width>69</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>380</y>
     <width>69</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_remitNameLbl">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Remittance Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_remitNameLineEdit">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>33</y>
     <width>133</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_orLbl">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>11</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>or</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_selectremitnalelbl">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>11</y>
     <width>170</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Remittance Name for Edit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_remitNameList">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>42</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
